src/prefect/cli/deployment.py

```diff
@@ -528,7 +529,7 @@ async def build(
     elif interval:
         schedule = IntervalSchedule(interval=timedelta(seconds=interval))
     elif rrule:
-        schedule = RRuleSchedule(rrule=rrule)
+        schedule = RRuleSchedule(**json.loads(rrule))
```

Comment:
I don't think this will work as intended?

The string needs to be in quotes?
```python
>>> json.loads('test')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/nix/store/k5j4ldd6l9fg35vfy64gahdd8nk9ifv8-python3-3.10.6/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/nix/store/k5j4ldd6l9fg35vfy64gahdd8nk9ifv8-python3-3.10.6/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/nix/store/k5j4ldd6l9fg35vfy64gahdd8nk9ifv8-python3-3.10.6/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
>>> json.loads('"test"')
'test'
```
Unpacking won't work for a string
```python
>>> {**json.loads('"test"')}
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'str' object is not a mapping
```

Reply:
Changing to always require a dictionary input is a breaking change. We should ensure this is backwards compatible with single strings.